Examples/Widgets/Utilities/SVGHelpers.js

```diff
@@ -0,0 +1,193 @@
+import {
```

Comment:
does it mean that if it is in the Examples folder it won't be exported in the ESM/UMD  module? 
If so, we might consider to keep it for a while to give time for VTK.js developers to adapt to the change.

Reply:
I don't think we should have it within vtk.js but we should definitely document/explain what is required if someone needs to reproduce that example. And mention, that is just an example on how to possibly do it but other framework like vue/react could be use to streamline such usage as well and that is beyond the scope of vtk.js to provide a svg infrastructure.

Reply:
I agree, documentation would be useful, the update example is not easy to read.

It would be worth mentioning the other (duplicate? ) SVG approach with `setDisplayCallback` (e.g. in `ImageCroppingWidget` example).

Reply:
I've updated the documentation in `concepts_widgets.md` and the comment block for `bindSVGRepresentation` to help guide folks who happen to use the SVGHelpers file.